src/app/components/Metadata/propTypesCheck.js

```diff
@@ -0,0 +1,17 @@
+import { checkPropTypes } from 'prop-types';
+
+const propTypeCheck = (props, componentName, propType) => {
+  const { type } = props;
+
+  const expectedType = {
+    [componentName]: propType,
+  };
+
+  if (type === 'article') {
```

Comment:
A small part of me wants to take this if statement out of this file and use it as a ternary in the component because we shouldn't have to call this function if the type is **not** article.

On the other hand, this looks much cleaner.